assets/js/src/core/modules/app/nav/index.ts

```diff
@@ -21,6 +21,12 @@ moduleSystem.registerModule({
   onInit: () => {
     const mainNavRegistryService = container.get<MainNavRegistry>(serviceIds.mainNavRegistry)
 
+    mainNavRegistryService.registerMainNavItem({
```

Comment:
I just moved the file menu to the top of the list so that it is the same for now like in the classic bundle ;)